### PR TITLE
Service delibery dashboard cap percentage values to 100% and present …

### PR DESCRIPTION
**Pull request by dmydlo**
…0/0 as 100 % (0 / 0)

Hi @calellowitz @Rohit25negi,
I have limited the percentage of `Home Visits` values in the Service Delivery Dashboard to 100%, right now `valid_visits == 5` and `expected_visits == 3` would be transformed into `100 % (3 / 3)` rather than to `166.66 % (5 / 3)`.
Also, rather than showing `(0 / 0)` right now `100% (0 / 0)` will be shown to maintain consistency with the AWW Performance Report.
These changes are related to JIRA [QA-339](https://dimagi-dev.atlassian.net/browse/QA-339) ticket.
Need QA: No
Environment: n/a
Locally Tested: Yes
Regards, Dawid

### CODE REVIEW

**Comment by Rohit25negi** [comment]
Looks good to merge once the Test passes

File: custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js
Comment (Rohit25negi) [reply custom/icds_reports/static/js/directives/service-delivery-dashboard/service-delivery-dashboard.directive.js]: Checked with heena. 100% is the required behaviour.

**Review by Rohit25negi — APPROVED** [review]
Looks good

File: custom/icds_reports/reports/service_delivery_dashboard.py
Comment (calellowitz) [inline custom/icds_reports/reports/service_delivery_dashboard.py]: I did not get a chance to review this before it was merged but this logic is wrong. It will lead to both undercounting of visits and inconsistent data at various levels. For example, imagine a sector with 2 AWWs, one who did 5/3 and one who did 1/3. At the AWW level, the AWW who did 5 visits will only have gotten credit for 3, but at the supervisor level it will say 6/6 even though the numbers shown at the lower level will not add up.

**Pull request merged**
